python/pyarrow/includes/libarrow_fs.pxd

```diff
@@ -301,6 +301,19 @@ cdef extern from "arrow/filesystem/api.h" namespace "arrow::fs" nogil:
         const CIOContext& io_context,
         int64_t chunk_size, c_bool use_threads)
 
+    CStatus HaveLibHdfs()
+
+    cdef enum HdfsDriver" arrow::io::HdfsDriver":
+        HdfsDriver_LIBHDFS" arrow::io::HdfsDriver::LIBHDFS"
+        HdfsDriver_LIBHDFS3" arrow::io::HdfsDriver::LIBHDFS3"
```

Comment:
I need to remove `HdfsDriver` as it is a leftover from: https://github.com/apache/arrow/commit/4e53749097ba687afd5e000067925def2e2802c9#diff-8f9209d74ff1ddfc6fed20ab199b6d158d0052382fae2c7bcef9f0bd96cebc95